iron-icon.html

```diff
@@ -134,20 +132,25 @@
          * precedence over a given icon attribute.
          */
         src: {
-          type: String,
-          observer: '_srcChanged'
+          type: String
         },
 
         /**
          * @type {!Polymer.IronMeta}
          */
         _meta: {
-          value: Polymer.Base.create('iron-meta', {type: 'iconset'}),
-          observer: '_updateIcon'
+          value: Polymer.Base.create('iron-meta', {type: 'iconset'})
         }
 
       },
 
+      observers: [
+        '_updateIcon(meta, isAttached)',
+        '_updateIcon(theme, isAttached)',
```

Comment:
I also don't know if this is ok, but `_updateIcon` is one of those evil observers that uses a lot of `this.` which may or may not be set in time in the future :(